src/response.rs

```diff
@@ -64,6 +65,24 @@ pub enum TxtRecordValue {
     Value(BString),
 }
 
+/// A Case-insensitive wrapper for key string of TXT record following spec's mandate:
+/// Case is ignored when interpreting a key,
+/// so "papersize=A4", "PAPERSIZE=A4", and "Papersize=A4" are all identical.
+#[derive(Eq, Clone, Debug, serde::Serialize, serde::Deserialize)]
+pub struct TxtRecordKey(String);
```

Comment:
We should probably add `#[serde(transparent)]` to the struct:

https://serde.rs/container-attrs.html#transparent

Reply:
Good idea! Pushed up the commit.